.tours/welcome-to-dreamhouse.tour

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "👋 Welcome to Dreamhouse",
+  "steps": [
+    {
+      "title": "Welcome",
+      "file": "force-app/main/default/lwc/propertySummary/propertySummary.html",
+      "line": 22,
+      "description": "Hello! Welcome to Dreamhouse (LWC) sample app. \n\nThis application incorporates Lightning web components and other Lightning Experience features to deliver a rich user experience that helps Dreamhouse, a fictitious real estate company, manage its properties and brokers using the Salesforce Platform.\n\nThis tour will show you around the repo at a high level. \n\nIn this file, you can see a simple `propertySummary` Lightning Web Component that displays a card with a short summary of a property (e.g. a house). Properties are stored as `Property__c` custom objects. In `propertySummary`, we use a [`lightning-record-form`](https://developer.salesforce.com/docs/component-library/bundle/lightning-record-form/documentation) base component to display the record information."
```

Comment:
Lightning web components follows a different case in this and the next step

Reply:
Good catch. I'm referring to components so I'll lower the case.